src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorSpectrum.cs

```diff
@@ -0,0 +1,1787 @@
+					//		case ColorSpectrumComponents.SaturationValue:
+					//			strongThis.m_hueRedSurface = minSurface;
+					//			strongThis.m_hueYellowSurface = CreateSurfaceFromPixelData(pixelWidth, pixelHeight, bgraMiddle1PixelData);
+					//			strongThis.m_hueGreenSurface = CreateSurfaceFromPixelData(pixelWidth, pixelHeight, bgraMiddle2PixelData);
+					//			strongThis.m_hueCyanSurface = CreateSurfaceFromPixelData(pixelWidth, pixelHeight, bgraMiddle3PixelData);
+					//			strongThis.m_hueBlueSurface = CreateSurfaceFromPixelData(pixelWidth, pixelHeight, bgraMiddle4PixelData);
+					//			strongThis.m_huePurpleSurface = maxSurface;
+					//			break;
+					//	}
+					//}
+					//else
+					{
+						WriteableBitmap minBitmap = ColorHelpers.CreateBitmapFromPixelData(pixelWidth, pixelHeight, bgraMinPixelData);
+						WriteableBitmap maxBitmap = ColorHelpers.CreateBitmapFromPixelData(pixelWidth, pixelHeight, bgraMaxPixelData);
+
+						switch (components2)
+						{
+							case ColorSpectrumComponents.HueValue:
+							case ColorSpectrumComponents.ValueHue:
+								strongThis.m_saturationMinimumBitmap = minBitmap;
+								strongThis.m_saturationMaximumBitmap = maxBitmap;
+								break;
+							case ColorSpectrumComponents.HueSaturation:
+							case ColorSpectrumComponents.SaturationHue:
+								strongThis.m_valueBitmap = maxBitmap;
+								break;
+							case ColorSpectrumComponents.ValueSaturation:
+							case ColorSpectrumComponents.SaturationValue:
+								strongThis.m_hueRedBitmap = minBitmap;
+								strongThis.m_hueYellowBitmap = ColorHelpers.CreateBitmapFromPixelData(pixelWidth, pixelHeight, bgraMiddle1PixelData);
+								strongThis.m_hueGreenBitmap = ColorHelpers.CreateBitmapFromPixelData(pixelWidth, pixelHeight, bgraMiddle2PixelData);
+								strongThis.m_hueCyanBitmap = ColorHelpers.CreateBitmapFromPixelData(pixelWidth, pixelHeight, bgraMiddle3PixelData);
+								strongThis.m_hueBlueBitmap = ColorHelpers.CreateBitmapFromPixelData(pixelWidth, pixelHeight, bgraMiddle4PixelData);
+								strongThis.m_huePurpleBitmap = maxBitmap;
+								break;
+						}
+					}
+
+					strongThis.m_shapeFromLastBitmapCreation = strongThis.Shape;
+					strongThis.m_componentsFromLastBitmapCreation = strongThis.Components;
+					strongThis.m_imageWidthFromLastBitmapCreation = minDimension;
+					strongThis.m_imageHeightFromLastBitmapCreation = minDimension;
+					strongThis.m_minHueFromLastBitmapCreation = strongThis.MinHue;
+					strongThis.m_maxHueFromLastBitmapCreation = strongThis.MaxHue;
+					strongThis.m_minSaturationFromLastBitmapCreation = strongThis.MinSaturation;
+					strongThis.m_maxSaturationFromLastBitmapCreation = strongThis.MaxSaturation;
+					strongThis.m_minValueFromLastBitmapCreation = strongThis.MinValue;
+					strongThis.m_maxValueFromLastBitmapCreation = strongThis.MaxValue;
+
+					strongThis.m_hsvValues = newHsvValues;
+
+					strongThis.UpdateBitmapSources();
+					strongThis.UpdateEllipse();
+				});
+			//});
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)